### DISCUSSION TITLE
Configure a external FIDO server for  doing this webauthn registration & authentication

### DISCUSSION THREAD

**Discussion by amitt0488** [OpenIdentityPlatform/OpenAM#646] [General]
I have a separate FIDO server which provides all the APIs for FIDO (Webauthn) registration & authentication
How can I configure that separate FIDO server URL in openAM module or any other place for doing this registration & authentication ?
https://github.com/OpenIdentityPlatform/OpenAM/wiki/How-to-Setup-WebAuthn-Authentication-in-OpenAM

**Comment by maximthomas** [comment]
I guess you should develop a custom authentication module for your FIDO server integration. https://github.com/OpenIdentityPlatform/OpenAM/wiki/How-to-Devleop-a-Custom-Authentication-Module
